README.rst

```diff
@@ -4,28 +4,36 @@ eventyay-tickets
 .. image:: https://codecov.io/gh/fossasia/eventyay-tickets/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/pretix/pretix
 
-
 Project status & release cycle
 ------------------------------
 
-Welcome to the eventyay project. The ticketing component of the system provides options for ticket sales and event related items such a Tshirts. Eventyay is developed since 2014. The ticketing component is based on a fork of pretix.
+Welcome to the **Eventyay** project! The ticketing component of the system provides options for **ticket sales and event-related items** such as T-shirts. Eventyay has been in development since **2014** and is based on a fork of **Pretix**.
```

Comment:
Thank you for your attempt. Please rephrase the "fork" clause. Only the ticketing component, not the whole Eventyay, is based on a fork of "Pretix".